.gitignore

```diff
@@ -2,3 +2,4 @@ _site
 .sass-cache
 .jekyll-metadata
 vendor
+.gitpod.yml
```

Comment:
This `.gitignore` change should be a separate pull request that can be discussed in isolation from this EIP.  I recommend removing it from this EIP (you can just do it in the GitHub UI by accepting this suggestion), and then submitting a separate PR that *only* adds `.gitpod.yml` to the ignore list so we can discuss it properly, and the history of `.gitignore` isn't cluttered with EIPs.
```suggestion
```

Reply:
Created #4865